todo_app/app.py

```diff
@@ -1,11 +1,22 @@
 from flask import Flask
+from flask import render_template
+from todo_app.data import session_items
+from flask import request, redirect, url_for
+
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/index', methods=['GET'])
```

Comment:
As above, by default I'd suggest leaving this as the `/` route, to avoid having to tell users where to navigate to find your homepage